cloudformation/services/concordia-service/service.yaml

```diff
@@ -31,6 +31,11 @@ Parameters:
         Type: String
         Default: /
 
+    PostgresHost:
```

Comment:
If we add the postgres host to the outputs, we could use `Fn::ImportValue` to retrieve it in this stack